weblab/accounts/tests/test_signals.py

```diff
@@ -25,3 +27,25 @@ def test_email_sent_on_user_creation(superuser):
     assert 'Test User' in body
     assert 'test@example.com' in body
     assert 'http://127.0.0.1:8000/admin/accounts/user/{}/change/'.format(user.pk) in body
+
+
+@pytest.mark.django_db
+def test_delete_user_directory(client, logged_in_user, my_dataset):
```

Comment:
Perhaps rename this test given we'll want to add some others:
```suggestion
def test_user_can_delete_own_account(client, logged_in_user, my_dataset):
```

Other tests could be:
* `test_delete_user_requires_login` (the original test you had, without the `logged_in_user`)
* `test_user_cannot_delete_other_account`